scripts/linux/generic-rust/install.sh

```diff
@@ -49,7 +49,7 @@ function install_rust()
     # but it's simpler to just always `update` whatever toolchain it is. `update` installs the toolchain
     # if it hasn't already been installed, so this also works for pinned versions.
     source $HOME/.cargo/env
-    rustup update "$(< "$PROJECT_ROOT/rust-toolchain")"
+    ( cd $PROJECT_ROOT; rustup show active-toolchain )
```

Comment:
not sure why this change?

Reply:
This is because I changed the toolchain override file to TOML, meaning its contents cannot be used directly as a command-line argument like before. As long as build invocations are run from each project root, this command should actually be unnecessary.

Reply:
The file moved one level up, and it's true that there's no need to keep the path in sync when `rustup` can look it up for us.

Reply:
`rustup show active-toolchain` coerces `rustup` into installing the toolchain specified in `rust-toolchain.toml` during the current build step rather than later on during a `cargo` invocation.